src/openfl/_internal/renderer/RenderSession.hx

```diff
@@ -76,16 +78,22 @@ class RenderSession {
 			if (gl.version == 2) {
 				
 				vaoContext = new VertexArrayObjectContext (gl);
+				instancedRendering = new GLInstancedRendering (gl);
 				
 			} else if (gl.version == 1) {
 				
 				var vertexArrayObjectsExtension = gl.getExtension ("OES_vertex_array_object");
-				
 				if (vertexArrayObjectsExtension != null) {
 					
 					vaoContext = new VertexArrayObjectExtension (vertexArrayObjectsExtension);
 					
 				}
```

Comment:
I know it's not your code, but can you make `VertexArrayObjectContext` implemented in the same way as your new `GLInstancedRendering`? I like the new code more and I would like to have a consistent way of creating such extension "wrappers" properly.